actions/create_document.lua

```diff
@@ -1,27 +1,29 @@
 event: ["request_post_document"]
 priority: 1
 
-local helpers = require("helpers")
+local content = require("content")
 
 -- POST /
 local post_uuid = uuid.v4()
-local file = io.open("content/" .. post_uuid, "w")
+local file_uuid = uuid.v4()
```

Comment:
what's the difference between `post_uuid` and `file_uuid`
I would think there should only be an `incoming_request_uuid` and a `document_uuid`